### ISSUE TITLE
remove useRef input focuses on render - accessibility

### ISSUE THREAD

**Issue by Athma-Vasi** [Athma-Vasi/MacAuley-Tech-Repair-Employee-Portal---frontend#15]
remove useRef input focuses on render on the form inputs, as the StepperWrapper steps ref should be the first focused elem on render, improving screen reader accessibility.

**Comment by Athma-Vasi** [comment]
done
